src/cr/cube/matrix/measure.py

```diff
@@ -1239,8 +1242,9 @@ def t_stats(self):
         Sa, Sb, Sab = self._selected_counts
         Na, Nb, Nab = self._valid_counts
         pa, pb, pab = Sa / Na, Sb / Nb, Sab / Nab
+        pct_a, pct_b = self._proportions
```

Comment:
i think that `pct` prefix should be changed in something that recall the proportions cause pct can be interpreted as percentages that is basically prop * 100.